Homework/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "13-npm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha -bdd test.js"
+  },
+  "dependencies": {
+    "mocha": "^8.4.0"
```

Comment:
This is also a `devDependency` since you use it just to run tests, not in your source code.